packages/blade/src/utils/makeBorderSize/makeBorderSize.test.ts

```diff
@@ -0,0 +1,17 @@
+import border from '../../tokens/global/border';
+import makeBorderSize from '.';
+
+describe('makeBorderSize', () => {
+  it('should return the border width in `px`', () => {
+    const space = makeBorderSize(border.width.thin);
+    expect(space).toEqual('1px');
+  });
+  it('should return the border radius in `px`', () => {
+    const space = makeBorderSize(border.radius.small);
+    expect(space).toEqual('2px');
+  });
+  it('should return the border radius in `%`', () => {
+    const space = makeBorderSize(border.radius.round);
+    expect(space).toEqual('50%');
+  });
```

Comment:
add a negative test case where we can check that `makeBorderSize('50%')` should never return `50px` or `makeBorderSize(50)` should never return `50`(without units)

Reply:
We already have test cases for both 

**Test A:** `expect(makeBorderSize(2)).toEqual('2px');`
and
**Test B:** `expect(makeBorderSize('50%')).toEqual('50%');`

If for any reason `makeBorderSize('50%')` returned `50px`, **Test B** would fail and similarly if for any reason `makeBorderSize(2)` returned `2`, **Test A** would fail

Not sure we we need negative test here?